src/components/InputandFilterPanel/InputandFilterPanel.styles.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "styled-components";
+import searchIcon from "../../images/search.svg";
+
+export const StyledInputandFilterSection = styled.section`
+	padding: 20px 40px;
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: start;
+	@media screen and (min-width: 550px) {
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+	}
+`;
+
+export const StyledForm = styled.form`
+	width: 100%;
+	@media screen and (min-width: 550px) {
+		width: 40%;
+	}
+`;
+
+export const StyledInputLabel = styled.label`
+	width: 100%;
+`;
+
+export const StyledInputElement = styled.input`
+	border: none;
+	width: 100%;
+	padding: 4% 0;
```

Comment:
this seems weird. Try to get rid of such padding

Reply:
fixed